src/Functional.Unions.FluentAssertions/Functional.Unions.FluentAssertions/Functional.Unions.FluentAssertions.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Authors>Ryan Marcotte + contributors</Authors>
+    <Version>2.1.0</Version>
```

Comment:
Should be version `1.0.0`, no?

Reply:
Oh yes, I think I was a little confused because I wasn't sure if the two packages had to match version numbers. I suppose other libraries like `Functional` only have sub packages match version numbers because there is an encompassing `Functional.All` package.

Reply:
Correct.  The `Functional` library is set up such that a version bump affects all packages.  These packages are independent, so their version numbers will vary independently.

Reply:
With that in mind, you can totally slap your name in the `Authors` space instead of mine 🙂